### PR TITLE
Board

### PR DESCRIPTION
**Pull request by lastmjs**
I've created the initial board component and gotten rid of the `bower.json` file and all dependencies except for johnny-five

### CODE REVIEW

**Comment by lastmjs** [comment]
@bdemann If you could look this over and add any comments, that would be great. Any problems in the code that you think should be addressed should be added as comments during the review process. Make sure to click submit review. I'll then address all of the comments and once everything is resolved I'll merge the pull request.

File: jfive-board.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
+<script>
+    class JFiveBoard extends HTMLElement {
+        constructor() {
+            super();
+
+            this.boardId;
```
Comment (bdemann) [inline jfive-board.html:6]: is there a JavaScript syntax to declare properties?

Comment (bdemann) [reply jfive-board.html:6]: not a problem

**Review by bdemann — APPROVED** [review]
It looks good!

**Pull request merged**